torch/onnx/symbolic_opset9.py

```diff
@@ -412,13 +412,13 @@ def _div_rounding_mode(g: jit_utils.GraphContext, self, other, rounding_mode):
 
 @_beartype.beartype
 def _trunc_divide(g: jit_utils.GraphContext, self, other):
-    out = g.op("Div", self, other)
+    quotient = g.op("Div", self, other)
     # the correct operation is truncate, which is not supported in ONNX,
     # we cannot call floor since it will behave differently for negative numbers
     # (eg. -0.1 should become -0 )
     # - if scalar_type information are not available, assume that
     # we need to call floor (treat as float)
-    out = g.op("Cast", out, to_i=_C_onnx.TensorProtoDataType.INT64)
+    casted = g.op("Cast", quotient, to_i=_C_onnx.TensorProtoDataType.INT64)
```

Comment:
TODO: Use cast to implement aten::trunc?

Reply:
Wonder if cast == trunc, for negative values, particularly.

Reply:
cast may overflow when the number is big, since float32 has a greater range iiuc